#! /bin/sh /usr/share/dpatch/dpatch-run
## 16-skip-apps.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## DP: do not compile some apps

@DPATCH@
--- ACE_wrappers.orig/apps/apps.mwc
+++ ACE_wrappers/apps/apps.mwc
@@ -2,4 +2,13 @@
 // $Id: apps.mwc 54744 2003-12-23 20:23:21Z dhinton $
 
 workspace {
+  exclude {
+    drwho
+    Gateway
+    JAWS
+    JAWS2
+    JAWS3
+    mkcsregdb
+    soreduce
+  }
 }
